src/black/handle_ipynb_magics.py

```diff
@@ -58,8 +58,11 @@ class Replacement:
 @lru_cache()
 def jupyter_dependencies_are_installed(*, verbose: bool, quiet: bool) -> bool:
     try:
-        import IPython  # noqa:F401
+        # isort: off
```

Comment:
```suggestion
        # tokenize_rt is less commonly installed than IPython, and IPython is expensive to import,
        # so import tokenize_rt first.
        # isort: off
```
So future readers know why we're doing this